janusgraph-dist/src/assembly/static/conf/gremlin-server/gremlin-server.yaml

```diff
@@ -2,8 +2,11 @@ host: 0.0.0.0
 port: 8182
 scriptEvaluationTimeout: 30000
 channelizer: org.apache.tinkerpop.gremlin.server.channel.WebSocketChannelizer
+graphManager: org.janusgraph.graphdb.management.JanusGraphManager
 graphs: {
-  graph: conf/gremlin-server/janusgraph-cassandra-es-server.properties}
+  graph: conf/gremlin-server/janusgraph-cassandra-es-server.properties,
+  janusConfigurationGraph: conf/janusgraph-cassandra-configurationgraph.properties
```

Comment:
Look at renaming to avoid "Janus" abbreviation. I don't think we've done that (yet).